local_groups/models.py

```diff
@@ -33,6 +33,21 @@ class Group(models.Model):
         blank=False,
         unique=True
     )
+
+    GROUP_TYPES = (
+        ('state-organizing-committee', 'State Organizing Committee'),
+        ('state-chapter', 'State Chapter'),
+        ('campus', 'Campus'),
+        ('local-group', 'Local Group')
+    )
+    group_type = models.CharField(
```

Comment:
Should this field be required? Not sure why it's optional.

Reply:
Also how are we going to populate the correct values in prod?

Reply:
Will populate manually, initially wasn't going to make required because if someone submitted a new group it would bork things, but working on this now.

Reply:
Is now required.